documentation/Who-Is-MrChecker/Test-Framework-Modules/Selenium-Test-Module-Selenium-UFT-Comparison.asciidoc

```diff
@@ -41,23 +41,23 @@ Java
 |Same as SE
 
 |Customizations
-|All is standard available
```

Comment:
What is that supposed to mean?

Reply:
The product is ready to use and not customizable